src/Coduo/PHPHumanizer/Resources/translations/number.en.yml

```diff
@@ -0,0 +1,42 @@
+hyphen: "-"
+conjunction: "and"
+separator: ","
+negative: "negative"
+decimal: "point"
+
+number:
+  zero: "zero"
```

Comment:
Maybe as a keys you could use numbers here? So it would be:

```yaml
number:
  0: "zero"
  1: "one"
  2: "two"
```
etc.  
This wuold allow us to remove $map field from Speller class.